[2023-12-20T13:19:35.061Z] info: docker.yaml was copied to the config folder
[2023-12-20T13:19:35.131Z] info: services.yaml was copied to the config folder
[2023-12-20T13:19:35.138Z] info: bookmarks.yaml was copied to the config folder
[2023-12-20T13:19:35.147Z] info: widgets.yaml was copied to the config folder
[2023-12-20T13:19:35.152Z] info: custom.css was copied to the config folder
[2023-12-20T13:19:35.156Z] info: custom.js was copied to the config folder
[2023-12-20T13:19:44.949Z] info: kubernetes.yaml was copied to the config folder
[2023-12-20T13:36:46.107Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:36:46.110Z] error: YAMLException: tab characters must not be used in indentation (10:5)

  7 |         href: http://192.168.124.14 ...
  8 |         description: 下载工具
  9 |     - casaOS:
 10 |     →href: http://192.168.124.14
----------^
 11 | →description: 管理工具
 12 | 
[2023-12-20T13:36:46.254Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:36:46.256Z] error: YAMLException: tab characters must not be used in indentation (10:5)

  7 |         href: http://192.168.124.14 ...
  8 |         description: 下载工具
  9 |     - casaOS:
 10 |     →href: http://192.168.124.14
----------^
 11 | →description: 管理工具
 12 | 
[2023-12-20T13:36:47.798Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:36:47.800Z] error: YAMLException: tab characters must not be used in indentation (10:5)

  7 |         href: http://192.168.124.14 ...
  8 |         description: 下载工具
  9 |     - casaOS:
 10 |     →href: http://192.168.124.14
----------^
 11 | →description: 管理工具
 12 | 
[2023-12-20T13:50:34.506Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:50:34.508Z] error: YAMLException: bad indentation of a mapping entry (12:9)

  9 |     - casaOS:
 10 |         href: http://192.168.124.14
 11 |         description: 管理工具
 12 |         - Portainer:
--------------^
 13 |             icon: portainer.png
 14 |             href: "http://192.168.124.1 ...
[2023-12-20T13:50:34.560Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:50:34.563Z] error: YAMLException: bad indentation of a mapping entry (12:9)

  9 |     - casaOS:
 10 |         href: http://192.168.124.14
 11 |         description: 管理工具
 12 |         - Portainer:
--------------^
 13 |             icon: portainer.png
 14 |             href: "http://192.168.124.1 ...
[2023-12-20T13:50:35.910Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:50:35.911Z] error: YAMLException: bad indentation of a mapping entry (12:9)

  9 |     - casaOS:
 10 |         href: http://192.168.124.14
 11 |         description: 管理工具
 12 |         - Portainer:
--------------^
 13 |             icon: portainer.png
 14 |             href: "http://192.168.124.1 ...
[2023-12-20T13:51:17.976Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T13:52:35.900Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T13:52:35.986Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:52:35.987Z] error: YAMLException: tab characters must not be used in indentation (32:5)

 29 |         href: http://192.168.124.10 ...
 30 |         description: 代码编写
 31 |     - Docker
 32 |     →server: iStoreOS
----------^
 33 |         widget:
 34 |             type: Docker
[2023-12-20T13:52:36.112Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:52:36.114Z] error: YAMLException: tab characters must not be used in indentation (32:5)

 29 |         href: http://192.168.124.10 ...
 30 |         description: 代码编写
 31 |     - Docker
 32 |     →server: iStoreOS
----------^
 33 |         widget:
 34 |             type: Docker
[2023-12-20T13:52:36.969Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:52:36.970Z] error: YAMLException: tab characters must not be used in indentation (32:5)

 29 |         href: http://192.168.124.10 ...
 30 |         description: 代码编写
 31 |     - Docker
 32 |     →server: iStoreOS
----------^
 33 |         widget:
 34 |             type: Docker
[2023-12-20T13:53:10.873Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:53:10.877Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: iStoreOS
--------------------^
 33 |         widget:
 34 |             type: Docker
[2023-12-20T13:53:11.810Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:53:11.812Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: iStoreOS
--------------------^
 33 |         widget:
 34 |             type: Docker
[2023-12-20T13:53:32.243Z] error: <httpProxy> Error calling http://192.168.0.124:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T13:53:32.258Z] error: <httpProxy> [
  500,
  Error: connect ETIMEDOUT 192.168.0.124:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -110,
    code: 'ETIMEDOUT',
    syscall: 'connect',
    address: '192.168.0.124',
    port: 9000
  }
]
[2023-12-20T13:53:32.264Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.0.124:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T13:55:40.586Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:55:40.590Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: iStoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:55:41.383Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:55:41.384Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: iStoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:56:09.432Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:56:09.433Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: iStoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:57:54.349Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:57:54.350Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: iStoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:57:55.107Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:57:55.109Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: iStoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:58:29.575Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:58:29.576Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: StoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:58:30.360Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:58:30.362Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: StoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:59:15.384Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T13:59:15.385Z] error: YAMLException: bad indentation of a sequence entry (32:15)

 29 |         href: http://192.168.124.101:8085/
 30 |         description: 代码编写
 31 |     - Docker
 32 |         server: StoreOS
--------------------^
 33 |         container: OneCloud
 34 | →
[2023-12-20T13:59:54.221Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T13:59:57.352Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T13:59:57.357Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:00:11.936Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:00:11.952Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:00:19.690Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:00:19.894Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:00:29.002Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:00:29.198Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:01:27.526Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:01:27.746Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:01:49.650Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:01:49.889Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:02:17.818Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:02:17.997Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:04:50.227Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:04:50.239Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:04:50.273Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T14:04:50.276Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-20T14:04:50.279Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T14:04:57.096Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:04:57.115Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T14:04:57.117Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-20T14:04:57.118Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T14:04:57.742Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:04:57.771Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T14:04:57.774Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-20T14:04:57.777Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T14:06:07.871Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T14:06:07.875Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-20T14:06:07.877Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T14:06:10.810Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:07:14.362Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:07:14.411Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T14:07:14.413Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-20T14:07:14.416Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T14:07:16.978Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T14:07:17.013Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T14:07:17.015Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-20T14:07:17.024Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T15:07:54.809Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:07:54.956Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-20T15:07:54.958Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-20T15:07:54.959Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-20T15:40:15.669Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:40:21.607Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:40:21.610Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:40:56.641Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:40:56.646Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:44:03.870Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:44:05.027Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:45:06.402Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:45:07.102Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:45:10.482Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:46:43.928Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:49:32.529Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:49:32.576Z] error: tab characters must not be used in indentation (34:1)

 31 |     - Docker:
 32 |         server: StoreOS
 33 |         container: OneCloud
 34 | →container: HomePage
------^
 35 | →
 36 | - D410c:
[2023-12-20T15:49:32.661Z] error: Failed to load services.yaml, please check for errors
[2023-12-20T15:49:32.662Z] error: YAMLException: tab characters must not be used in indentation (34:1)

 31 |     - Docker:
 32 |         server: StoreOS
 33 |         container: OneCloud
 34 | →container: HomePage
------^
 35 | →
 36 | - D410c:
[2023-12-20T15:50:16.135Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:50:16.867Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:58:47.777Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:58:48.145Z] error: <credentialedProxyHandler> HTTP Error 404 calling http://192.168.124.14:9000/api/endpoints/1/docker/containers/json?all=1
[2023-12-20T15:58:48.669Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:58:48.902Z] error: <credentialedProxyHandler> HTTP Error 404 calling http://192.168.124.14:9000/api/endpoints/1/docker/containers/json?all=1
[2023-12-20T15:59:18.058Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T15:59:18.245Z] error: <credentialedProxyHandler> HTTP Error 404 calling http://192.168.124.14:9000/api/endpoints/1/docker/containers/json?all=1
[2023-12-20T16:00:36.519Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T16:00:37.567Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T16:01:51.817Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-20T16:01:55.802Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T02:56:37.396Z] error: tab characters must not be used in indentation (27:1)

 24 |             key: ptr_/WFI4DVzzD ...
 25 |     - kodbox:
 26 |         server: OneCloud
 27 | →container: kodbox-kodbox-1
------^
 28 | 
 29 | - J3060:
[2023-12-21T02:56:37.656Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T02:56:37.663Z] error: YAMLException: tab characters must not be used in indentation (27:1)

 24 |             key: ptr_/WFI4DVzzD ...
 25 |     - kodbox:
 26 |         server: OneCloud
 27 | →container: kodbox-kodbox-1
------^
 28 | 
 29 | - J3060:
[2023-12-21T02:56:39.939Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T02:56:39.945Z] error: YAMLException: tab characters must not be used in indentation (27:1)

 24 |             key: ptr_/WFI4DVzzD ...
 25 |     - kodbox:
 26 |         server: OneCloud
 27 | →container: kodbox-kodbox-1
------^
 28 | 
 29 | - J3060:
[2023-12-21T02:56:43.086Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T02:56:43.087Z] error: YAMLException: tab characters must not be used in indentation (27:1)

 24 |             key: ptr_/WFI4DVzzD ...
 25 |     - kodbox:
 26 |         server: OneCloud
 27 | →container: kodbox-kodbox-1
------^
 28 | 
 29 | - J3060:
[2023-12-21T02:57:17.684Z] error: <credentialedProxyHandler> HTTP Error 401 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-21T02:57:29.447Z] error: <credentialedProxyHandler> HTTP Error 401 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-21T06:06:40.575Z] error: tab characters must not be used in indentation (59:1)

 56 |         container: HomePage
 57 |     - Portainer:
 58 |         description: 容器情况
 59 | →widget:
------^
 60 |             type: portainer
 61 |             url: http://192.168 ...
[2023-12-21T06:06:41.055Z] error: tab characters must not be used in indentation (59:1)

 56 |         container: HomePage
 57 |     - Portainer:
 58 |         description: 容器情况
 59 | →widget:
------^
 60 |             type: portainer
 61 |             url: http://192.168 ...
[2023-12-21T06:06:41.546Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T06:06:41.548Z] error: YAMLException: tab characters must not be used in indentation (59:1)

 56 |         container: HomePage
 57 |     - Portainer:
 58 |         description: 容器情况
 59 | →widget:
------^
 60 |             type: portainer
 61 |             url: http://192.168 ...
[2023-12-21T06:07:23.895Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T06:07:23.897Z] error: YAMLException: bad indentation of a mapping entry (61:17)

 58 |         description: 容器情况
 59 |         widget:
 60 |         icon: portainer.png
 61 |             type: portainer
----------------------^
 62 |             url: http://192.168.124.14:9000
 63 |             env: 3
[2023-12-21T06:07:24.398Z] error: bad indentation of a mapping entry (61:17)

 58 |         description: 容器情况
 59 |         widget:
 60 |         icon: portainer.png
 61 |             type: portainer
----------------------^
 62 |             url: http://192.168.124.14:9000
 63 |             env: 3
[2023-12-21T06:07:24.611Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T06:07:24.612Z] error: YAMLException: bad indentation of a mapping entry (61:17)

 58 |         description: 容器情况
 59 |         widget:
 60 |         icon: portainer.png
 61 |             type: portainer
----------------------^
 62 |             url: http://192.168.124.14:9000
 63 |             env: 3
[2023-12-21T06:08:47.835Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T06:08:47.836Z] error: YAMLException: bad indentation of a mapping entry (61:17)

 58 |         description: 容器情况
 59 |         widget:
 60 |         icon: portainer.png
 61 |             type: portainer
----------------------^
 62 |             url: http://192.168.124.14:9000
 63 |             env: 3
[2023-12-21T06:09:16.575Z] error: <credentialedProxyHandler> HTTP Error 404 calling http://192.168.124.14:9000/api/endpoints/3/docker/containers/json?all=1
[2023-12-21T07:12:07.982Z] error: tab characters must not be used in indentation (40:1)

 37 |         container: navidrome
 38 |         widget:
 39 |             type: navidrome
 40 | →    url: http://192.168.124.14 ...
------^
 41 | →    user: wood
 42 | →    token: b1e4999163950f96fa7 ...
[2023-12-21T07:12:07.988Z] error: tab characters must not be used in indentation (40:1)

 37 |         container: navidrome
 38 |         widget:
 39 |             type: navidrome
 40 | →    url: http://192.168.124.14 ...
------^
 41 | →    user: wood
 42 | →    token: b1e4999163950f96fa7 ...
[2023-12-21T07:12:08.088Z] error: tab characters must not be used in indentation (40:1)

 37 |         container: navidrome
 38 |         widget:
 39 |             type: navidrome
 40 | →    url: http://192.168.124.14 ...
------^
 41 | →    user: wood
 42 | →    token: b1e4999163950f96fa7 ...
[2023-12-21T07:12:08.250Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T07:12:08.251Z] error: YAMLException: tab characters must not be used in indentation (40:1)

 37 |         container: navidrome
 38 |         widget:
 39 |             type: navidrome
 40 | →    url: http://192.168.124.14 ...
------^
 41 | →    user: wood
 42 | →    token: b1e4999163950f96fa7 ...
[2023-12-21T07:12:08.349Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T07:12:08.351Z] error: YAMLException: tab characters must not be used in indentation (40:1)

 37 |         container: navidrome
 38 |         widget:
 39 |             type: navidrome
 40 | →    url: http://192.168.124.14 ...
------^
 41 | →    user: wood
 42 | →    token: b1e4999163950f96fa7 ...
[2023-12-21T07:12:10.142Z] error: Failed to load services.yaml, please check for errors
[2023-12-21T07:12:10.144Z] error: YAMLException: tab characters must not be used in indentation (40:1)

 37 |         container: navidrome
 38 |         widget:
 39 |             type: navidrome
 40 | →    url: http://192.168.124.14 ...
------^
 41 | →    user: wood
 42 | →    token: b1e4999163950f96fa7 ...
[2023-12-21T08:35:12.148Z] error: <dockerStatusService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/status/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T08:35:12.195Z] error: <dockerStatusService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/status/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T08:35:12.281Z] error: <dockerStatusService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/status/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T08:35:12.790Z] error: <dockerStatsService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/stats/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T08:35:13.037Z] error: <dockerStatsService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/stats/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T08:35:13.059Z] error: <dockerStatsService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/stats/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T08:35:13.079Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:13.097Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:13.125Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:13.165Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:13.190Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:13.211Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:16.000Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:16.038Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:35:16.042Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:38:59.368Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:38:59.371Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:38:59.444Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:30.467Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:30.470Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:30.477Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:34.099Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:34.107Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:38.499Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:38.502Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:38.539Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:47.237Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:47.247Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:48.772Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:48.779Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:51.372Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:52.520Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:52.575Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:40:52.577Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:41:09.112Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:41:09.117Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:41:09.221Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:41:19.747Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:41:19.753Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:41:19.758Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:41:20.789Z] error: <dockerStatsService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:43:06.874Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:43:07.093Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T08:43:07.096Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.101:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-21T09:14:05.711Z] error: <dockerStatusService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/status/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T09:14:05.741Z] error: <dockerStatusService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/status/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-21T09:14:05.797Z] error: <dockerStatusService> TypeError: Cannot read properties of null (reading 'conn')
    at handler (/app/.next/server/pages/api/docker/status/[...service].js:89:86)
    at Object.apiResolver (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/api-utils/node.js:366:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NextNodeServer.runApi (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:481:9)
    at async Object.fn (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:741:37)
    at async Router.execute (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/router.js:252:36)
    at async NextNodeServer.run (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:365:29)
    at async NextNodeServer.handleRequest (/app/node_modules/.pnpm/next@12.3.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/base-server.js:303:20)
    at async Server.<anonymous> (/app/server.js:19:5)
[2023-12-24T14:24:42.364Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:42.434Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:42.438Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:42.468Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:42.522Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:42.526Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:47.816Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:48.095Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-24T14:24:48.099Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-24T14:24:48.102Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-24T14:24:48.144Z] error: <httpProxy> Error calling http://192.168.124.14:8080/api/v2/torrents/info...
[2023-12-24T14:24:48.150Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:8080
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 8080
  }
]
[2023-12-24T14:24:48.153Z] error: <qbittorrentProxyHandler> HTTP 500 getting data from qBittorrent.  Data: { error: [Object] }
[2023-12-24T14:24:48.215Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:48.257Z] error: <httpProxy> Error calling http://192.168.124.14:9091/transmission/rpc...
[2023-12-24T14:24:48.259Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9091
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9091
  }
]
[2023-12-24T14:24:48.260Z] error: <transmissionProxyHandler> Error getting data from Transmission: 500.  Data: { error: [Object] }
[2023-12-24T14:24:48.264Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:48.295Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:48.298Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:48.336Z] error: <httpProxy> Error calling http://192.168.124.14:4533/rest/getNowPlaying...
[2023-12-24T14:24:48.338Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:4533
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 4533
  }
]
[2023-12-24T14:24:48.343Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:24:48.389Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/4/docker/containers/json...
[2023-12-24T14:24:48.391Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-24T14:24:48.393Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/4/docker/containers/json?all=1
[2023-12-24T14:24:48.913Z] error: <qbittorrentProxyHandler> Error logging in to qBittorrent: Data: Fails.
[2023-12-24T14:24:54.462Z] error: <qbittorrentProxyHandler> Error logging in to qBittorrent: Data: Fails.
[2023-12-24T14:27:22.074Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:27:22.201Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json...
[2023-12-24T14:27:22.203Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-24T14:27:22.203Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/2/docker/containers/json?all=1
[2023-12-24T14:27:22.275Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:27:22.279Z] error: <httpProxy> Error calling http://192.168.124.14:8080/api/v2/torrents/info...
[2023-12-24T14:27:22.280Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:8080
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 8080
  }
]
[2023-12-24T14:27:22.282Z] error: <qbittorrentProxyHandler> HTTP 500 getting data from qBittorrent.  Data: { error: [Object] }
[2023-12-24T14:27:22.323Z] error: <httpProxy> Error calling http://192.168.124.14:9091/transmission/rpc...
[2023-12-24T14:27:22.324Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9091
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9091
  }
]
[2023-12-24T14:27:22.325Z] error: <transmissionProxyHandler> Error getting data from Transmission: 500.  Data: { error: [Object] }
[2023-12-24T14:27:22.327Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:27:22.329Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:27:22.359Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:27:22.361Z] error: <dockerStatusService> Error: connect ECONNREFUSED 192.168.124.14:2375
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16)
[2023-12-24T14:27:22.376Z] error: <httpProxy> Error calling http://192.168.124.14:4533/rest/getNowPlaying...
[2023-12-24T14:27:22.376Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:4533
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 4533
  }
]
[2023-12-24T14:27:22.421Z] error: <httpProxy> Error calling http://192.168.124.14:9000/api/endpoints/4/docker/containers/json...
[2023-12-24T14:27:22.422Z] error: <httpProxy> [
  500,
  Error: connect ECONNREFUSED 192.168.124.14:9000
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1555:16) {
    errno: -111,
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '192.168.124.14',
    port: 9000
  }
]
[2023-12-24T14:27:22.423Z] error: <credentialedProxyHandler> HTTP Error 500 calling http://192.168.124.14:9000/api/endpoints/4/docker/containers/json?all=1
[2023-12-24T14:27:22.850Z] error: <qbittorrentProxyHandler> Error logging in to qBittorrent: Data: Fails.
